src/app/utils/route-utils.ts

```diff
@@ -7,6 +7,33 @@ import { Network } from '../../types/network'
 import { SearchScope } from '../../types/searchScope'
 import { isStableDeploy } from '../../config'
 import { getSearchTermFromRequest } from '../components/Search/search-utils'
+import { isLayerHidden } from '../../types/layers'
+import { isTesting } from './env'
+
+export const fixedNetwork = isTesting
+  ? undefined
+  : (process.env.REACT_APP_FIXED_NETWORK as Network | undefined)
+export const fixedLayer = isTesting ? undefined : (process.env.REACT_APP_FIXED_LAYER as Layer | undefined)
```

Comment:
Why is this needed? Why would we want to unset fixed network/layer for testing only?

Reply:
Because we want to use simple routes and paths for the tests, and don't want to rewrite them for complicated, conditional logic. 

It's easiest to always run the tests against the basic configuration, independently of how we plan to deploy the app.

In fact, _not_ unsetting for playwright also breaks _those_ tests, which is still an unsolved issue. (How do we easily detect that we are running inside playwrighet?)